packages/react-components/react-toolbar/src/stories/Toolbar.stories.tsx

```diff
@@ -5,6 +5,9 @@ import bestPracticesMd from './ToolbarBestPractices.md';
 
 export { Default } from './ToolbarDefault.stories';
 export { Small } from './ToolbarSmall.stories';
+export { WithTooltip } from './ToolbarWithTooltip.stories';
+export { WithPopover } from './ToolbarWithPopover.stories';
+export { Subtle } from './ToolbarSubtle.stories';
```

Comment:
could you also please add best practices according to https://fluentsite.z22.web.core.windows.net/0.62.0/components/toolbar/definition - not sure if all are relevant to v9, but some are.

Reply:
Done